tools/utils/download_blender.sh

```diff
@@ -64,20 +73,64 @@ function get_extractor() {
     echo "${extractor}"
 }
 
+function verify_download_integrity() {
+    local version=${1}
+    local target_filepath=${2}
+
+    if [ ! -f "${target_filepath}" ]; then
+        return 1
+    fi
+
+    echo "Found Blender ${ver} download. Verifying the integrity."
+
+    local target_filename checksum_url checksum_url download_dir
+    target_filename="$(basename "${target_filepath}")"
+    download_dir="$(dirname "${target_filepath}")"
+    checksum_url="${BLENDER_CHECKSUM_URL[${ver}]}"
+    checksum_filename="$(basename "${checksum_url}")"
+
+    pushd "${download_dir}" 1> /dev/null
+
+    curl --fail -s "${checksum_url}" -o "${checksum_filename}"
```

Comment:
Is it possible to remove .md5 file after checking the checksum?

Reply:
I thought at first it would be nice to keep them but we download it anyway again, so why not remove it. Let me change this.